File: packages/sdk-py/src/grantex/manifests/banking_aa.py
from grantex.manifest import ToolManifest, Permission

manifest = ToolManifest(
    connector="banking_aa",
    description="Banking Account Aggregator API",
    tools={
        "fetch_bank_statement": Permission.READ,
        "check_account_balance": Permission.READ,
        "get_transaction_list": Permission.READ,
        "request_consent": Permission.WRITE,
        "fetch_fi_data": Permission.READ,
    },
)

File: packages/sdk-ts/src/manifests/banking_aa.ts
import { ToolManifest, Permission } from '../manifest.js';

export const bankingAaManifest = new ToolManifest({
  connector: 'banking_aa',
  description: 'Banking Account Aggregator API',
  tools: {
    fetch_bank_statement: Permission.READ,
    check_account_balance: Permission.READ,
    get_transaction_list: Permission.READ,
    request_consent: Permission.WRITE,
    fetch_fi_data: Permission.READ,
  },
});
